src/diffs/package-json.js

```diff
@@ -0,0 +1,37 @@
+// Heavily inspired by https://github.com/greenkeeperio/greenkeeper/blob/8be4bfc45dc991ec0c3331ac15b688728f5938a5/lib/diff-package-json.js
+
+const _ = require('lodash')
```

Comment:
Oops - guess it's a dependency of almost everything so I didn't notice I didn't install it explicitly. Good catch